sites/docs/pages/components/accordion.md

```diff
@@ -3,40 +3,69 @@ title: Accordion
 sidebar_position: 1
 ---
 
-<img src="/img/accordion.png" alt="accordion" width="600"/>
+<Accordion>
+  <AccordionItem title="Item 1">
+
+    This is the first item's accordion body.
+
+    You can use **markdown** here too!
+
+    Make sure to include an empty line after the component if you want to use markdown.
+
+  </AccordionItem>
+  <AccordionItem title="Item 2">
+
+    This is the second item's accordion body with <b>bold text</b>.
+
+  </AccordionItem>
+  <AccordionItem title="Item 3">
+
+    This is the third item's accordion body.
+
+  </AccordionItem>
+</Accordion>
+
 
 ```markdown
 <Accordion>
   <AccordionItem title="Item 1">
+
     This is the first item's accordion body.
+
+    You can use **markdown** here too!
+    Make sure to include an empty line after the component if you want to use markdown.
+
   </AccordionItem>
   <AccordionItem title="Item 2">
+
     This is the second item's accordion body with <b>bold text</b>.
+
   </AccordionItem>
   <AccordionItem title="Item 3">
+
     This is the third item's accordion body.
+
   </AccordionItem>
 </Accordion>
 ```
 
+## Examples
+
 ## Options
 
 ### Accordion
 
-<PropListing name="single" options={['true', 'false']}>
-
-When true, only a single accordian item can be open at once.
-
-</PropListing>
+  <PropListing
+      name="single"
+      description="When true, only a single accordian item can be open at once."
```

Comment:
`PropListing` expects description content in a slot. 

I think if you replace line 37 in `PropListing` with: 

`<div id="markdown-slot"><slot > {description} </slot> </div>`

It'll work with both the description prop and the slot.